scapy/layers/tls/handshake.py

```diff
@@ -135,7 +136,7 @@ def __init__(self, name, default, dico, length_from=None, itemfmt="!H"):
         self.itemsize = struct.calcsize(itemfmt)
         i2s = self.i2s = {}
         s2i = self.s2i = {}
-        keys = dico.keys()
+        keys = list(dico.keys())
         for k in keys:
```

Comment:
I meant `iteritems` as there is `dico[k]` in the loop so it could be accessed using items instead of keys but I am not aware of any performance impact between the two.
But if we use keys only, `for k in dico:` would be cleaner IMO